plyer/platforms/android/qrbarcodereader.py

```diff
@@ -0,0 +1,21 @@
+from jnius import cast
+from jnius import autoclass
+from plyer.facades import QrBarcodeReader
+
+PythonActivity = autoclass('org.renpy.android.PythonActivity')
+Intent = autoclass('android.content.Intent')
+Uri = autoclass('android.net.Uri')
+
+class AndroidQrBarcodeReader(QrBarcodeReader):
+
+    def _scan(self):
+
+        intent = Intent()
+        intent.setAction(Intent.ACTION_VIEW)
+        intent.setData(Uri.parse('http://www.onlinebarcodereader.com'))
```

Comment:
Also, [this](http://www.ama-dev.com/iphone-qr-code-library-ios-7/) might be interesting too. Seems like iphone has something built-in. PyObjus could help probably?